airlock/business_logic.py

```diff
@@ -853,6 +853,14 @@ def check_status(
                     f"Cannot set status to {to_status.name}; request has unapproved files."
                 )
 
+            if (
+                to_status == RequestStatus.APPROVED
+                and not len(release_request.all_files_set()) > 0
```

Comment:
I think this should probably check `output_files_set` rather than `all_files_set` to check for at least one output file. We don't want to allow releasing of a request that only has supporting files.

We also shouldn't need to check the file count, just the truthiness of `release_request.output_files_set()`

Reply:
good points, done